regen/feature.pl

```diff
@@ -76,6 +76,17 @@ BEGIN
 ###########################################################################
 # More data generated from the above
 
+if (keys %feature > 32) {
+    die "cop_features only has room for 32 features";
```

Comment:
What would be the thing to do if we run out of bits? Can we easily turn the field into a U64, or would we make a second one?

Reply:
If U64 can be used, it may make sense to reuse `cop_hints` (it's a U32, but 26 of its bits are currently in use)

Reply:
Simplest for working with feature.pm/${^FEATURE_BITS} would probably be a U8 array, but that's difficult to write a SAVEFEATUREBITS() for.

We could make it a pointer as for warnings, but since the feature bits are fixed size and will be present in any cop, the pointer is just overhead.

But for now we have over twice the feature bits as features, so I don't feel much urgency to decide right now.

Reply:
> If U64 can be used, it may make sense to reuse `cop_hints` (it's a U32, but 28 of its bits are currently in use)

If we worked through $^H we'd need to require -Duse64bitint to safely update it from feature.pm